README.md

```diff
@@ -134,12 +134,13 @@ Try `ocr-transform -h` to get an overview:
 
 <!-- BEGIN-EVAL echo '<pre>';./bin/ocr-transform.sh -h 2>&1;echo '</pre>'  -->
 <pre>
-Usage: ocr-transform [-dhL] <from> <to> [<infile> [<outfile>]] [-- <script-args>]
+Usage: ocr-transform [-dhLv] <from> <to> [<infile> [<outfile>]] [-- <script-args>]
```

Comment:
I suggest to run the version command here as well before showing the usage.

Reply:
bump

Reply:
Why? Typically programs don't show the version when the uses requests help. We might show the version in case of program errors.

Reply:
Okay, I checked some other CLI tools and you are right (also there are exceptions, e.g. unzip).